product.json

```diff
@@ -84,7 +84,7 @@
 	"builtInExtensions": [
 		{
 			"name": "Microsoft.sqlservernotebook",
-			"version": "0.4.0",
+			"version": "0.4.2",
```

Comment:
Is there a reason you chose 0.4.2 for this? If this is adding support for new CUs then I would say we should bump to 0.5.0 (and leave revision bumps for bug fixes)

Reply:
Not a specific reason, I was following the same format that we used for previous versions. I'll make the change.